File: estudo/neri/algoritmo/c/Prestacao.c
#include <stdio.h>
int main(){    
    float prest, valor_prest, taxa, dias_atraso;
    printf("Informe o valor da prestacao: ");
    scanf("%f",&valor_prest);
    printf("Informe a taxa de juros: ");
    scanf("%f",&taxa);
    printf("Informe a quantidade de dias em atraso: ");
    scanf("%f",&dias_atraso);
    prest = valor_prest + (valor_prest * (taxa /100) * dias_atraso);
    printf("O valor corrigido da prestacao ficou de: = %f",prest);
}

File: estudo/neri/algoritmo/java/Prestacao.java
import javax.swing.JOptionPane;

public class Prestacao {
public static void main(String args[]){
    float prest, valor_prest, taxa, dias_atraso;
    valor_prest = Float.parseFloat(JOptionPane.showInputDialog("Informe o valor da prestacao: "));
    taxa = Float.parseFloat(JOptionPane.showInputDialog("Informe a taxa de juros: "));
    dias_atraso = Float.parseFloat(JOptionPane.showInputDialog("Informe os dias em atraso: "));
    prest = valor_prest + (valor_prest * (taxa /100) * dias_atraso);
    JOptionPane.showMessageDialog(null,"O valor corrigido da prestacao ficou de: = "+prest);
}    
}
